test/common/plugins/otel_metrics/kibana.jsonc

```diff
@@ -2,6 +2,8 @@
   "type": "plugin",
   "id": "@kbn/open-telemetry-instrumented-plugin",
```

Comment:
I don't know if we should still own this plugin and if we still need it as it should be part of the @elastic/stack-monitoring  as I saw the [PR](https://github.com/elastic/kibana/pull/135999) adding it. 
cc: @smith 

But anyway thanks for the change!